Nos/Views/ThreadRootView.swift

```diff
@@ -1,12 +1,34 @@
 import SwiftUI
 
+/// Displays a reply note below its root note.
 struct ThreadRootView<Reply: View>: View {
+
+    /// The root note.
     var root: Event
+
+    /// Whether the root note is interactive.
+    var allowsHitTestingOnRoot: Bool
```

Comment:
The fact that the doc comment doesn't have anything in common with the property name makes me think a different name would be more appropriate:
```suggestion
    var isRootNoteInteractive: Bool
```